File: Module/Home/src/main/java/com/module/home/game/view/IQuickGameView3.java
package com.module.home.game.view;

import com.component.busilib.friends.SpecialModel;
import com.module.home.model.GameKConfigModel;
import com.module.home.model.SlideShowModel;

import java.util.List;

public interface IQuickGameView3 {
    void setBannerImage(List<SlideShowModel> slideShowModelList);

//    void setRecommendInfo(List<RecommendModel> list);

    void setQuickRoom(List<SpecialModel> list, int offset);

//    void setGameConfig(GameKConfigModel gameKConfigModel);

    void showTaskRedDot(boolean show);
}

File: Module/Home/src/main/java/com/module/home/game/view/IQuickGameView3.kt
package com.module.home.game.view

import com.component.busilib.model.PartyRoomInfoModel
import com.component.person.model.UserRankModel
import com.component.busilib.banner.SlideShowModel

interface IQuickGameView3 {
    fun setBannerImage(slideShowModelList: List<SlideShowModel>?)

    fun showTaskRedDot(show: Boolean)

    fun setRegionDiff(model: UserRankModel?)

    fun setPartyRoomList(list: List<PartyRoomInfoModel>?)
}
